crd/nodeinfo/manifests/acn.azure.com_nodeinfoes.yaml

```diff
@@ -0,0 +1,52 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.11.3
+  creationTimestamp: null
+  name: nodeinfoes.acn.azure.com
+spec:
+  group: acn.azure.com
+  names:
+    kind: NodeInfo
+    listKind: NodeInfoList
+    plural: nodeinfoes
```

Comment:
"info" is mass noun, it is already plural.

Reply:
oh, i just saw the previous comment that this was autogen...that's annoying

Reply:
yes dont see a way of fixing this

Reply:
updated to nodesInfo

Reply:
nodesinfo doesn't make any sense

Reply:
hmm. just sticking to nodeInfo now